ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/tabularBlock/tabularBlock.component.scss

```diff
@@ -0,0 +1,77 @@
+$grid-gap: 0; //10px;
+$border-color: #1d1d1d;
+
+.tabular-container {
+    border: 1px solid $border-color;
+    border-bottom: none;
+    width: 50%;
+
+    .headers {
+        grid-gap: $grid-gap;
+        font-weight: bold;
+        font-size: 1.2rem;
+
+        .header {
+            grid-row: 1;
+        }
+    }
+
+    .questions {
+        &--row {
+            grid-gap: $grid-gap;
+
+            .question-title {
+                padding-top: 40px;
+            }
+        }
+
+        &--cell {
+            position: relative;
+
+            .separator {
+                position: absolute;
+                top: 50%;
+                right: -10px;
+            }
+        }
+    }
+
+    .cell {
+        padding: 15px;
+        border-right: 1px solid $border-color;
+        border-bottom: 1px solid $border-color;
+
+        + .cell {
+            border-right: none;
+        }
+
+    }
+}
+
+.tabular-container.composite {
+    border: none;
+    width: 70%;
+
+    * {
+        border: none;
+    }
+
+    .questions--row {
```

Comment:
is this double dash intended?

Reply:
Actually not. 
There was no any special sense there. 
Fixed.